test/cli/samples/merge-deprecations/rollup.config.js

```diff
@@ -0,0 +1,17 @@
+var replace = require( 'rollup-plugin-replace' );
+
+let warnings = [];
+
+module.exports = {
+	entry: 'main.js',
+	input: 'main.js',
+	format: 'es',
+	dest: '_actual/bundle1.js',
+	output: {
+		file: '_actual/bundle1.js',
```

Comment:
Perhaps make this `_actual/bundle2.js` so that we are testing the precedence here for future changes too.

Reply:
With the current implementation, `dest: bundle1.js` would replace `output.file: `bundle2.js` (as it is currently the case for all deprecations, i.e. the deprecated config takes precedence). I guess you would rather expect `output.file` to win?

I could change that, but then I would try to change it for all deprecations at once. Should I do that?

Reply:
It does seem to make sense that the newer form should have higher precedence.

In theory it's a breaking change, but it could be worthwhile.